config/envs/default.py

```diff
@@ -21,7 +22,7 @@ class DefaultConfig:
     TEST_FUND_STORE_API_HOST = "fund_store"
     TEST_ACCOUNT_STORE_API_HOST = "account_store"
     TEST_NOTIFICATION_SERVICE_HOST = "notification_service"
-    USE_LOCAL_DATA = False
+    USE_LOCAL_DATA = strtobool(environ.get("USE_LOCAL_DATA", "False"))
```

Comment:
heres a description of what strtobool does: https://docs.python.org/3/distutils/apiref.html#:~:text=distutils.util.strtobool(val)%C2%B6

The environ.get tries to grab USE_LOCAL_DATA from the env (.flaskenv locally) and if it is not set it will default to "False". USE_LOCAL_DATA is used in a switch when fetching data to decide whether to call local data (within the repository) or remote data via an endpoint get.

Reply:
thank you! ❤️